Centaurus.Domain/helpers/ClientExceptionHelper.cs

```diff
@@ -9,17 +9,21 @@ public static class ClientExceptionHelper
     {
         public static ResultStatusCodes GetExceptionStatusCode(Exception exception)
         {
-            var resultStatusCode = ResultStatusCodes.InternalError;
```

Comment:
What's the point of introducing custom exceptions if you don't encapsulate the logic there?

```c#
public class ForbiddenException: BaseClientException
{
    public override ResultStatusCodes StatusCode get => ResultStatusCodes.Forbidden;
}
```

Then you can avoid `switch-case` and simply do 

```c#
if (exception is BaseClientException) return (exception as BaseClientException).StatusCode;
return ResultStatusCodes.InternalError;
```

Reply:
We need this custom exception class to handle client exceptions. 
About StatusCode property: there was some troubles with project dependencies in the earlier versions. I need to investigate if we can implement this approach now. Or we can use two enums and mapping technique